README.md

```diff
@@ -1,22 +1,16 @@
 # IDOM &middot; [![Tests](https://github.com/idom-team/idom/workflows/Test/badge.svg?event=push)](https://github.com/idom-team/idom/actions?query=workflow%3ATest) [![PyPI Version](https://img.shields.io/pypi/v/idom.svg)](https://pypi.python.org/pypi/idom) [![License](https://img.shields.io/badge/License-MIT-purple.svg)](https://github.com/idom-team/idom/blob/main/LICENSE)
 
-IDOM is a Python web framework for building **interactive websites without needing a
-single line of Javascript**. These sites are built from small elements of functionality
-like buttons text and images. IDOM allows you to combine these elements into reusable
-"components" that can be composed together to create complex views.
-
-Ecosystem independence is also a core feature of IDOM. It can be added to existing
-applications built on a variety of sync and async web servers, as well as integrated
-with other frameworks like Django, Jupyter, and Plotly Dash. Not only does this mean
-you're free to choose what technology stack to run on, but on top of that, you can run
-the exact same components wherever you need them. For example, you can take a component
-originally developed in a Jupyter Notebook and embed it in your production application
-without changing anything about the component itself.
+IDOM is a Python framework for building **interactive websites without needing a single line of JavaScript**. Web elements you provide (ex. `button`, `div`, `image`) are combined into reusable "components". Components [can be nested together](https://idom-docs.herokuapp.com/docs/guides/creating-interfaces/your-first-components/index.html#conditional-rendering) and utilize [ReactJS-style hooks](https://idom-docs.herokuapp.com/docs/reference/hooks-api.html) to create complex web pages. Ultimately, all IDOM components are communicated to the browser and then rendered by ReactJS.
```

Comment:
I feel like this paragraph is trying to say too much. I don't expect people to understand everything about IDOM from this README since they should go to the actual documentation for more details. In some sense this should be a teaser.

Reply:
My idea behind this is to highlight the main selling points of IDOM more prominently. From the perspective of people who don't know about IDOM, I find these features to be more important than ecosystem independence.

The last sentence is the one I find over-excessive, but I wanted to make sure we mention we are ReactJS-backed somewhere in the readme.

Reply:
I'll see if I can wordsmith this to be more compact.

Reply:
Sorry, I've been heads down on [this PR](https://github.com/idom-team/idom/pull/703) so haven't managed to think about this yet.

Reply:
I've significantly reduced the verbosity while still highlighting the key features.

Take a look at the live hosted version to see what it looks like now:
https://github.com/Archmonger/idom/tree/readme-summary-revise